infra/resources/prod/redis.tf

```diff
@@ -46,49 +46,49 @@ module "redis_messages" {
 }
 
 
-module "com_redis_messages" {
-  source = "github.com/pagopa/terraform-azurerm-v4//redis_cache?ref=v1.2.1"
+resource "azurerm_redis_cache" "com_redis_messages" {
+  name = "${local.project}-${local.domain}-redis-01"
```

Comment:
use the `resource_name` function to name this resource, [as I suggested here](https://github.com/pagopa/io-messages/pull/260#discussion_r2115500495)